tdi/python/influxSignal.py

```diff
@@ -125,10 +158,26 @@ def influxSignal(fieldKey, where, series=None, database=None, config=None, shotS
     if len(whereList) > 0:
         where = 'WHERE %s' % (' AND '.join(whereList),)
 
+    groupBy = ''
+    if deltaTime is not None:
+        # Convert deltaTime from seconds to milliseconds, if deltaTime is less than 1ms this will break
+        deltaTimeMS = int(deltaTime * 1000)
+        groupBy = 'GROUP BY time(%sms)' % (deltaTimeMS,) # The trailing 'ms' is for milliseconds
+        # If the fieldKey does not have an aggregation function, Influx will error out
+        # Attempt to wrap the field key with an aggregation function, but don't affect
+        # any equations stored in it
+        # For example:
+        #   10*fVal will become 10*MEAN(fVal)
```

Comment:
It turns out that you can't do math unless there's an aggregation (I believe), so we could potentially simplify this section and make it just wrap the fieldKey without the regex